src/manager.js

```diff
@@ -376,15 +378,43 @@ AdManager.prototype.loadAds = function(element) {
       window.index_headertag_lightspeed.slotDisplay(thisEl.id, ads);
     }
     thisEl.setAttribute('data-ad-load-state', 'loading');
+
+    if (slot.eagerLoad) {
+      this.refreshSlots([slot], ads);
+    }
   }
+};
 
+/**
+ * Fetches a new ad for just single dom element
+ *
+ * @param {domElement} DOM element containing the DFP ad slot
+ * @returns undefined
+*/
+AdManager.prototype.refreshSlot = function(domElement) {
+  var slot = this.slots[domElement.id];
+  var ads = this.findAds(domElement);
 
-  if (slotsToLoad.length > 0) {
-    if (typeof window.index_headertag_lightspeed === 'undefined') {
-      this.googletag.pubads().refresh(slotsToLoad);
-    } else {
-      window.index_headertag_lightspeed.slotRefresh(slotsToLoad, ads);
-    }
+  if (slot) {
+    this.refreshSlots([slot], ads);
+  }
+};
+
+/**
+ * Fetches a new ad for each slot passed in
+ *
+ * @param {slotsToLoad} One or many slots to fetch new ad for
+ * @returns undefined
+*/
+AdManager.prototype.refreshSlots = function (slotsToLoad, ads) {
+  if (slotsToLoad.length == 0) {
+    return;
+  }
+
+  if (typeof window.index_headertag_lightspeed === 'undefined') {
+    this.googletag.pubads().refresh(slotsToLoad, { changeCorrelator: false });
```

Comment:
`changeCorrelator` is important to set here.  Since we're now making separate ad requests, preserving the correlator (which stays the same for 30 seconds in between ad calls), it treats them as one ad call on the back-end which helps preserve the ability for roadblock sponsorships.